scripts/requirements-base.txt

```diff
@@ -14,6 +14,7 @@ PyYAML>=6.0
 pykwalify
 
 # used by west_commands
+bflb-mcu-tool
```

Comment:
Yes, it can but not all users are using this platform. So, we opted to provide a message when you run west flash. If the tool is not installed you get an error from bflb_mcu_tool.py:

```python
raise ValueError("Couldnt find bflb-mcu-tool in PATH, please install with pip install bflb-mcu-tool")
```